cloudinit/sources/DataSourceOVF.py

```diff
@@ -76,7 +78,7 @@ def _get_data(self):
         ud = ""
         vd = ""
         vmwareImcConfigFilePath = None
-        nicspath = None
+        nicsPath = None
```

Comment:
generally python is snake_case, not camelCase, i don't think the change here actually helps anything.

Reply:
Revert it back, thanks.